NEWS.md

```diff
@@ -1,3 +1,28 @@
+# infer 0.3.1.9000
+
+## Deprecation changes
+
+- Deprecated `p_value()` (use `get_p_value()` instead) (#180).
+- Deprecated `conf_int()` (use `get_confidence_interval()` instead) (#180).
+- Deprecated (via warnings) plotting p-value and confidence interval in `visualize()` (use new functions `shade_p_value()` and `shade_confidence_interval()` instead) (#178).
+
+## New functions
+
+- `shade_p_value()` - {ggplot2}-like layer function to add information about p-value region to `visualize()` output. Has alias `shade_pvalue()`.
+- `shade_confidence_interval()` - {ggplot2}-like layer function to add information about confidence interval region to `visualize()` output. Has alias `shade_ci()`.
```

Comment:
Is there a reason why these functions have abbreviated aliases but the two above(`get_p_value()` and `get_confidence_interval()`) do not?

Reply:
Actually, it looks like these analogous aliased were created (which is  good idea, I think), they're just not noted here.

Reply:
Those aliases were added in v0.3.0: https://github.com/tidymodels/infer/blob/master/NEWS.md